todo.js

```diff
@@ -0,0 +1,26 @@
+const addTaskBtn = document.getElementById("addTask");
+const taskInputText = document.getElementById("taskInput");
+const taskList = document.getElementById("taskList");
+
+let todos = [];
+
+const resetTaskInput = () => {
+  taskInputText.value = "";
+};
+const createTask = () => {
+  taskList.innerHTML = "";
+  todos.forEach((todo) => {
+    const newTask = document.createElement("li");
+    newTask.innerHTML = `${todo}`;
+    taskList.appendChild(newTask);
+  });
```

Comment:
```suggestion
  taskList.innerHTML = "";
  
  todos.forEach((todo) => {
    const newTask = document.createElement("li");
    newTask.innerHTML = `${todo}`;
    taskList.appendChild(newTask);
  });
```

Reply:
Would you please explain why we need an empty line at line no. 12?

Reply:
Just to make the code a little bit nicer. A small separation between a iteration and an assignment